src/testing/platform/testing-host-ref.ts

```diff
@@ -44,8 +33,8 @@ export const registerInstance = (
     hostRef = getHostRef(elm);
   }
 
+  lazyInstance.__stencil__getHostRef = () => hostRef;
   hostRef.$lazyInstance$ = lazyInstance;
-  return hostRefs.set(lazyInstance, hostRef);
```

Comment:
Not sure if the return value as weakmap is somewhere used, i tried to check the code but couldnt find any references which depends on the return value.

Reply:
Yeah, I think it doesn't have any impact.